### PR TITLE
Context aware functionality updates

### PR DESCRIPTION
**Pull request by mfdunlap**
This makes the code functional when running run.py (a bug from the prev commit) and updated the test code in utils.py

### CODE REVIEW

File: flaskSpellChecker/utils.py [hunk 1]
```diff
@@ -206,6 +206,7 @@ def rankCorrections(dictionary, possibilities, prevWord="", nextWord=""):
 
 
 if __name__ == "__main__":
```
Comment (lorenzopappone) [inline flaskSpellChecker/utils.py:208]: Are we sure that we want to make utils.py runnable? Can you motivate this choice?

File: flaskSpellChecker/utils.py [hunk 2]
```diff
@@ -214,20 +215,27 @@ def rankCorrections(dictionary, possibilities, prevWord="", nextWord=""):
...
-    misspellings, index = spellCheckText(ga, text)
-    print("Mispellings:", misspellings)
-    print("Indices:", index)
+    #misspellings, index = spellCheckText(ga, text)
+    #print("Mispellings:", misspellings)
+    #print("Indices:", index)
+
+    while (text != "exit"):
+        text = str(input("Enter text to spell check (Irish): "))
+        misppellings, indicies = spellCheckText(ga, text)
```
Comment (lorenzopappone) [inline flaskSpellChecker/utils.py:238]: Can you correct the variables' names?

**Review by lorenzopappone — COMMENTED** [review]
Thanks for the updates, just be sure to merge with develop branch and not with the main one.

File: flaskSpellChecker/utils.py (hunk 1 above)
Comment (mfdunlap) [reply flaskSpellChecker/utils.py:208]: This is just internal test code to make sure that the functions in utils.py work the way they're supposed to.

File: flaskSpellChecker/utils.py (hunk 2 above)
Comment (mfdunlap) [reply flaskSpellChecker/utils.py:238]: This just allows for us to spell check infinite cases without having to run the code multiple times. And we just type "exit" to stop testing and exit the code.

**Comment by mfdunlap** [comment]
I changed the merge to the develop branch instead of the main branch.

**Pull request merged**